6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,29 @@
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
+
+//process
+
+const args = process.argv;
+
+function average(num1, num2, num3) {
+  const avg = (num1 + num2 + num3) / 3;
+  console.log(avg);
+}
+average(Number(args[2]), Number(args[3]), Number(args[4]));
+
+//minimist
+
+const minimist = require("minimist");
```

Comment:
It's a good practice to put all the import statements in the top of the file. It makes the code more readable :)
So I will suggest to move this line up.